### DISCUSSION TITLE
Heizkörperthermostate: Externe Temperaturvorgabe

### DISCUSSION THREAD

**Discussion by Simonh98** [SukramJ/aiohomematic#739] [Q&A]
Ist es möglich, die Ist-Temperatur der Thermostate z.B. über Kalibrierwerte zu beeinflussen oder einen externen Wert vorzugeben?

**Comment by Baxxy13** [comment]
Man könnte die IST-Temperatur über den Offset (bis +/- 3,5°C) beeinflussen.
Ist aber nicht für häufige Anpassungen geeignet da der Offset-Wert ins Gerät geschrieben werden muss. Es wird also (jedesmal) eine Config übertragen und die Schreibzyklen des eingebauten Speichers sind endlich.

Für externe Werte muss ein Wandthermostat passend zur Gerätefamilie (HM/HmIP) direktverknüpft werden.
Ne andere Möglichkeit gibt es nicht.
